pages.pl/common/http.md

```diff
@@ -1,32 +1,36 @@
 # http
 
-> HTTPie: HTTP client, ma być łatwiejszy w użyciu niż cURL.
+> HTTPie: klient HTTP przeznaczony do testowania, debugowania i ogólnej interakcji z API i serwerami HTTP.
 > Więcej informacji: <https://httpie.io/docs/cli/usage>.
 
-- Pobierz adres URL do pliku:
+- Wykonaj proste żądanie GET (wyświetla nagłówki odpowiedzi i zawartość):
 
-`http --download {{przyklad.org}}`
+`http {{https://example.org}}`
 
-- Wyślij dane zakodowane w formularzu:
+- Wyświetl podane części treści (`H`: nagłówki żądania, `B`: ciało żądania, `h`: nagłówki odpowiedzi, `b`: ciało odpowiedzi, `m`: metadane odpowiedzi):
```

Comment:
```suggestion
- Wyświetl podane części treści (`H`: nagłówki żądania, `B`: treść żądania, `h`: nagłówki odpowiedzi, `b`: treść odpowiedzi, `m`: metadane odpowiedzi):
```
Probably both ways are correct(I am not sure), but I never seen "body" in this context translated to "ciało".
https://sjp.pwn.pl/sjp/cialo;2448948.html
https://www.diki.pl/slownik-angielskiego?q=body